tests/mc_comm/test_mc_comm.py

```diff
@@ -1134,3 +1148,194 @@ def test_McCommunicationProcess__raises_error_if_status_beacon_not_received_in_a
     )
     with pytest.raises(SerialCommStatusBeaconTimeoutError):
         invoke_process_run_and_check_errors(mc_process)
+
+
+def test_McCommunicationProcess__processes_reboot_command(
+    four_board_mc_comm_process_no_handshake, mantarray_mc_simulator, mocker
+):
+    mc_process = four_board_mc_comm_process_no_handshake["mc_process"]
+    board_queues = four_board_mc_comm_process_no_handshake["board_queues"]
+    simulator = mantarray_mc_simulator["simulator"]
+    input_queue = board_queues[0][0]
+    output_queue = board_queues[0][1]
+
+    mocker.patch.object(
+        mc_simulator,
+        "_get_secs_since_reboot_command",
+        autospec=True,
+        side_effect=[MAX_MC_REBOOT_DURATION_SECONDS / 2],
```

Comment:
note for future---this is a bit of a magic number at the moment. having just reviewed the code, I know this relates to the "typical reboot time" of the simulator, but it may be good for clarity in the future to create this as a separate variable (possibly in constants.py, or maybe just in simulator.py and have the test suite import it from there)